drivers/docker/driver.go

```diff
@@ -946,6 +970,10 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 
 			publishedPorts[containerPort+"/tcp"] = getPortBinding(network.IP, hostPortStr)
 			publishedPorts[containerPort+"/udp"] = getPortBinding(network.IP, hostPortStr)
+			if nebulaIpAddr != nil {
+				publishedPorts[containerPort+"/tcp"] = append(publishedPorts[containerPort+"/tcp"], getPortBinding(*nebulaIpAddr, hostPortStr)[0])
```

Comment:
for these, you can also just do
```golang
append(publishedPorts[containerPort+"/tcp"], getPortBinding(*nebulaIpAddr, hostPortStr)...)
```
e.g., use the spread (`...`) operator, instead of relying on taking the `[0]` index.

should be functionally equivalent... but less likely to have errors if something weird happens.